freqtrade/strategy/interface.py

```diff
@@ -134,6 +136,39 @@ def __init__(self, config: dict) -> None:
         self._last_candle_seen_per_pair: Dict[str, datetime] = {}
         super().__init__(config)
 
+        # Gather informative pairs from @informative-decorated methods.
+        self._ft_informative: Dict[
+            Tuple[str, str], Tuple[InformativeData,
+                                   Callable[[Any, DataFrame, dict], DataFrame]]] = {}
+        for attr_name in dir(self.__class__):
+            cls_method = getattr(self.__class__, attr_name)
+            if not callable(cls_method):
+                continue
+            ft_informative = getattr(cls_method, '_ft_informative', [])
```

Comment:
Wouldn't you want to use "None" here as default argument?

That would make the "next" line skip this immediately (if it falls into default, it's not relevant for the remainder of this function) ?